### PR TITLE
Update web-components: 3.0.24 --> 3.0.26

### PR DESCRIPTION
**Pull request by joshbuchea**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3970,10 +3970,10 @@
   dependencies:
     defer-to-connect "^2.0.1"
 
-"@terraware/web-components@^3.0.24":
-  version "3.0.24"
-  resolved "https://registry.yarnpkg.com/@terraware/web-components/-/web-components-3.0.24.tgz#2fe4156de220db5c447dc09ff0f7475ca0fe1650"
-  integrity sha512-lMJEa69BtI0uqS6l04+WbnCEL8BuPlTE9BEpk1zmI65/HDI9if3orL8MRsR9u8GmA9I6yl+3B2BG7XNFOXo86w==
+"@terraware/web-components@^3.0.26":
+  version "3.0.27"
```
Comment (constanzauanini) [inline yarn.lock:3974]: Why does it says 27 here??

Comment (joshbuchea) [reply yarn.lock:3974]: Oops I didn't catch that, thanks!

**Review by constanzauanini — APPROVED** [review]

**Comment by joshbuchea** [comment]
Not planning to merge this PR, just doing a quick sanity check to confirm the previous two updates in `web-components` still pass the e2e tests. Pretty sure I broke the tests with my recent changes in `web-components` for `3.0.27`, just confirming.

**Comment by joshbuchea** [comment]
Ok I assumed it was my changes that broke the tests, always nice to confirm though

**Pull request closed without merging**